pkgs/development/python-modules/pythonparser/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "pythonparser";
+  version = "1.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qby6w2pcfwxi4w62rly08d9wj32amf6m7rmbrqw0xz6mirq7rrp";
+  };
+  LC_ALL = "en_US.UTF-8";
+
+  propagatedBuildInputs = [
+    regex
+  ];
+
+  # Tests pass, but test cleanup is broken.
```

Comment:
Please elaborate? Usually we can possibly fix this during review.

Reply:
✓Fixed breakage and enabled tests.